test/test.tf

```diff
@@ -6,6 +6,31 @@ module "newrelic_monitoring" {
   newrelic_app_name                 = var.newrelic_app_name
   newrelic_fully_qualified_app_name = var.newrelic_fully_qualified_app_name
 
+  response_status_codes_alerts = [
```

Comment:
I'd consider using `status_code_alerts` as it is shorter and still carries the same meaning.

Also, can this be expressed as a list of blocks:

```
module "newrelic_monitoring" {
...

status_code_alert {
  name = "..."
  ...
}

status_code_alert {
  name = "..."
  ...
}
```

Reply:
Unfortunately this is not possible 😞